test/rules/CloudFront.test.ts

```diff
@@ -397,4 +399,29 @@ describe('Amazon CloudFront', () => {
       validateStack(stack, ruleId, TestType.COMPLIANCE);
     });
   });
+
+  describe('CloudFrontDefaultRootObjectConfigured: CloudFront distributions require a default object', () => {
+    const ruleId = 'CloudFrontDefaultRootObjectConfigured';
+
+    // Non Compliance
+    test('Noncompliance ', () => {
+      new Distribution(stack, 'rDistribution', {
```

Comment:
Prefixing resources with `r` is a  habit that I had carried over when transitioning from CloudFormation to CDK, slowly getting rid of all of these  😅)

```suggestion
      new Distribution(stack, 'Distribution', {
```